src/Layer.ts

```diff
@@ -75,7 +75,7 @@ namespace pixi_display {
                         renderer.renderTexture.clear(group.clearColor as any);
                     }
                 }
-                renderer.texture.unbind(rt);
+                renderer.texture.unbind(rt.baseTexture);
```

Comment:
Not sure if this works, but it should be BaseTexture according to `unbind` types.

Reply:
But maybe types in pixijs are incorrect, it was `PIXI.Texture|PIXI.BaseTexture` in commits history.